CHANGES.md

```diff
@@ -8,16 +8,27 @@ To be released.
 
 ### Backward-incompatible API changes
 
+ -  Removed `LiteDBStore` class.  Use `DefaultStore` instead.  [[#662]]
+
 ### Backward-incompatible network protocol changes
 
 ### Backward-incompatible storage format changes
 
+ -  The introduced `DefaultStore` is incompatible with the existed `LiteDBStore`
```

Comment:
Could you explain the adjustment you suggested?

Reply:
I thought <q>existing</q> is the correct form. I referenced https://dictionary.cambridge.org/dictionary/english/incompatible.

Reply:
I chose the past tense because `LiteDBStore` is gone now.  Would it be incorrect?  On the other hand, the sentences in the example you linked do not imply the existing things are gone.

Reply:
If you mean that, how about this?

```
The introduced `DefaultStore' is incompatible at the file-level with the `LiteDBStore` that existed.
```

Reply:
Fixed: feb6fca3c055c92fa3b8c233a3a071b457ecf8d8.